Docs/Notes.md

```diff
@@ -0,0 +1,11 @@
+# Notes
+
+## UIRefreshControl + viewport-fit=cover bug
+
+To scale the viewport of a browser to fill the display, you may have set viewport-fit=cover;
+`<meta name="viewport" content="viewport-fit=cover">`.
+
+If a view `allowsPullToRefresh` in a turbo-ios native app with `viewport-fit=cover`, 
+the UIRefreshControl will not be positioned as expected.
+
+You can fix this by conditionally removing `viewport-fit=cover` when loading in a native app.
```

Comment:
Do you have a screenshot for this issue?

We set the viewport to `width=device-width,initial-scale=1,maximum-scale=1,viewport-fit=cover` and haven't noticed this issue.

Reply:
There could be confounders at play.  Would need to repro on a minimal app.  But here are screenshots of our app (deployment target 14.0, running on 15.4.1) where the only difference is the inclusion of `viewport-fit=cover`:

Without `viewport-fit=cover`:
![IMG_4551](https://user-images.githubusercontent.com/68957/162977411-a65fca7a-0f9e-498b-ab27-2e67d243aa35.PNG)

With `viewport-fit=cover`:
![IMG_4550](https://user-images.githubusercontent.com/68957/162977416-4a75dcc7-0b0c-4e4a-8f7e-6eb905b10e9b.PNG)

Reply:
Ahh you're right ours does that too, now I look at it. I was always like 🤷  that's high up but whatever.